examples/src/main/scala/org/jtech/drone/ws/AsciiServiceActor.scala

```diff
@@ -24,11 +24,13 @@ class AsciiServiceActor extends Actor {
       println(s"Unhandled message $err")
   }
 
-  def broadcast(w: WsMessage): Unit =
-    if (lastSend < System.currentTimeMillis()) {
-      lastSend += 200
+  def broadcast(w: WsMessage): Unit = {
+    val currentTime = System.currentTimeMillis
+    if (currentTime - lastTime > 32) {
```

Comment:
Fix attempt at 4cfeb73d8cd2f07c169da6e2f6e883af7141beaa

And again, it's not throttling (slowing down but keeping all messages, which will make the feed appear in slow motion) but frame limiting (dropping frames to achieve a desired framerate).